server/test/repository/AccountRepositoryTest.java

```diff
@@ -362,6 +362,43 @@ public void findApplicantsWithIncorrectDobPath() {
     assertThat(applicants.get(0).getApplicantData().getApplicantName().get()).isEqualTo("Bar");
   }
 
+  @Test
+  public void findApplicantsNeedingPrimaryApplicantInfoDataMigration() {
+    // First name only
+    ApplicantModel applicant = savePlainApplicant();
```

Comment:
suggestion (non-blocking): I find this test a little hard to read. Would it work to split up the applicants into separate applicants to make it clear they're not the same person? And then also test that the applicants returned are the ones you expect (based on ids or whatever else).